pytorch_lightning/utilities/model_helpers.py

```diff
@@ -69,3 +74,19 @@ def is_overridden(
     parent_code = parent_attr.__code__
 
     return instance_code != parent_code
+
+
+def get_model_size(model: nn.Module) -> int:
+    """
+    Calculates the size of a nn.Module in bytes by saving the model to a temprorary file
+    and reading in the size.
+
+    Returns:
+        Number of megabytes in the parameters of the input module
+    """
+    # TODO: Implement a method without needing to download the model
+    tmp_name = f"{uuid.uuid4().hex}.pt"
+    torch.save(model.state_dict(), tmp_name)
+    size_mb = os.path.getsize(tmp_name) / 1e6
+    os.remove(tmp_name)
+    return size_mb
```

Comment:
IMO this makes more sense here instead of `model_helpers.py`: https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/utilities/memory.py